deployments/goerli/usdc/deploy.ts

```diff
@@ -21,37 +13,14 @@ async function deployContracts(deploymentManager: DeploymentManager, deploySpec:
   const ethers = deploymentManager.hre.ethers;
   const signer = await deploymentManager.getSigner();
 
-  // Deploy governance contracts
-  const { COMP, fauceteer, timelock } = await cloneGov(deploymentManager);
-
-  const usdcProxyAdmin = await deploymentManager.deploy('USDC:admin', 'vendor/proxy/transparent/ProxyAdmin.sol', []);
-  const usdcImpl = await deploymentManager.clone('USDC:implementation', clone.usdcImpl, []);
-  const usdcProxy = await deploymentManager.clone('USDC', clone.usdcProxy, [usdcImpl.address]);
-  const usdcProxyAdminSlot = '0x10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b';
-  const USDC = usdcImpl.attach(usdcProxy.address);
-
-  await deploymentManager.idempotent(
-    async () => !sameAddress(await ethers.provider.getStorageAt(usdcProxy.address, usdcProxyAdminSlot), usdcProxyAdmin.address),
-    async () => {
-      trace(`Changing admin of USDC proxy to ${usdcProxyAdmin.address}`);
-      trace(await wait(usdcProxy.connect(signer).changeAdmin(usdcProxyAdmin.address)));
+  // Declare existing assets as aliases
+  const COMP = await deploymentManager.existing('COMP', '0x3587b2F7E0E2D6166d6C14230e7Fe160252B0ba4');
+  const USDC = await deploymentManager.existing('USDC', '0x07865c6E87B9F70255377e024ace6630C1Eaa37F');
+  const WBTC = await deploymentManager.existing('WBTC', '0xAAD4992D949f9214458594dF92B44165Fb84dC19');
+  const WETH = await deploymentManager.existing('WETH', '0x42a71137C09AE83D8d05974960fd607d40033499');
```

Comment:
Should these be declared before `await mintTokens(deploymentManager);`?

Reply:
`COMP` is tied to `cloneGov`, the others could be moved but kinda nice to have them all together?

Reply:
Actually I misread, for some reason I thought `mintTokens` was being called before these declarations